iphone/Maps/UI/EditBookmark/EditTrackViewController.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  EditTrackViewController.swift
+//  OMaps
+//
+//  Created by Ivan Koshkin on 10.7.21..
+//  Copyright © 2021 Organic Maps. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class EditTrackViewController: MWMTableViewController {
+  private enum Sections: Int {
+    case info
+    case delete
+    case count
+  }
+  
+  private enum InfoSectionRows: Int {
+    case title
+    //case color // TODO: need to know which layer color should be changed in track
+    //case lineWidth // TODO: possible new section & ability - edit track line width
+    case bookmarkGroup
+    case count
+  }
+  
+  private var editingCompleted: (Bool) -> Void
+
+  private let trackId: MWMTrackID
+  private var trackTitle: String?
+  private var trackGroupTitle: String?
+  private var trackGroupId = FrameworkHelper.invalidCategoryId()
+  //private var trackColor: BookmarkColor // or add TrackColor type
+
+  private let bookmarksManager = BookmarksManager.shared()
+  
+  init(trackId: MWMTrackID, editCompletion completion: @escaping (Bool) -> Void) {
+    self.trackId = trackId
+    
+    let bm = BookmarksManager.shared()
+    let track = bm.track(withId: trackId)
+    
+    trackTitle = track.trackName
+    //trackColor = ....
+
+    let category = bm.category(forTrackId: trackId)
+    trackGroupId = category.categoryId
+    trackGroupTitle = category.title
+    
+
+    editingCompleted = completion
+
+    super.init(style: .grouped)
+  }
+  
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    title = L("track").capitalized
```

Comment:
Also "track" translations?

Reply:
I've added "track_title" to strings.txt (it is more explicit for this case).